packages/@aws-cdk/aws-stepfunctions-tasks/lib/run-glue-job-task.ts

```diff
@@ -0,0 +1,116 @@
+import * as glue from '@aws-cdk/aws-glue';
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Duration } from '@aws-cdk/core';
+import { getResourceArn } from './resource-arn-suffix';
+
+/**
+ * Properties for RunGlueJobTask
+ */
+export interface RunGlueJobTaskProps {
+
+  /**
+   * The service integration pattern indicates different ways to start the Glue job.
+   *
+   * The valid value for Glue is either FIRE_AND_FORGET or SYNC.
+   *
+   * @default FIRE_AND_FORGET
+   */
+  readonly integrationPattern?: sfn.ServiceIntegrationPattern;
+
+  /**
+   * The ID of a previous JobRun to retry.
+   *
+   * @default - Creates new run
+   */
+  readonly jobRunId?: string;
+
+  /**
+   * The job arguments specifically for this run. For this job run, they replace the
+   * default arguments set in the job definition itself.
+   *
+   * @default - Only arguments consumed by Glue
+   */
+  readonly arguments?: { [key: string]: string };
+
+  /**
+   * The number of AWS Glue data processing units (DPUs) to allocate to this JobRun.
+   *
+   * @default - 10
+   */
+  readonly allocatedCapacity?: number;
+
+  /**
+   * The JobRun timeout in minutes. This is the maximum time that a job run can consume
+   * resources before it is terminated and enters TIMEOUT status.
+   *
+   * @default - 2,880 (48 hours)
```

Comment:
> when should a dash be used between @default and the value? Only when the default value isn't an actual typescript value?

Correct. Typically, a dash is used when you want to describe the default (as text) rather than specify a specific value.

> Will specifying the service defaults here confuse users?

Probably, yes. I would phrase it as `@default - uses the same as configured on the glue job` or something along those lines.